custom-widget-builder/src/CustomTagHandler.ts

```diff
@@ -125,12 +127,27 @@ export class CustomTagHandler {
     this.constraints = new PropertyConstraint(min, max);
   }
 
-  private removeFirstAndLastChar(str: string): string {
+  private static removeFirstAndLastChar(str: string): string {
       return str.substring(1, str.length-1);
   }
 
-  private removeQuotes(str: string) {
+  private static removeQuotes(str: string) {
```

Comment:
Why static here? I guess it's only called in this class no?

Reply:
See my global comment